### PR TITLE
Clarified that these examples are Mule 3. Cleaned up.

### PR DESCRIPTION
**Pull request by hannanelson**
Fixes DOCS-6211 and DOCS-5356.

### CODE REVIEW

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 1]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-CloudHub supports building HTTPS- and SSL-based services so you can be sure of your data confidentiality.
```
Comment (jorgegarciamule) [inline modules/ROOT/pages/building-an-https-service.adoc]: CloudHub supports "deploying", "running" or similar

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 2]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-CloudHub supports building HTTPS- and SSL-based services so you can be sure of your data confidentiality. 
-The configuration steps shown in this document are meant for deploying your Mule app to CloudHub.
+CloudHub supports building HTTPS- and SSL-based services so you ensure your data confidentiality.
```
Comment (jorgegarciamule) [inline modules/ROOT/pages/building-an-https-service.adoc]: CloudHub supports "deploying", "running" or similar

**Review by jorgegarciamule — APPROVED** [review]
I trust that the snippets work

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 3]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc:9]: INSTRUCTIONS FOR EDIT:
a medium (1 topic) clean-up/clarification PR. Green only. EOD Tuesday?

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 4]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-CloudHub supports building HTTPS- and SSL-based services so you can be sure of your data confidentiality. 
-The configuration steps shown in this document are meant for deploying your Mule app to CloudHub.
+CloudHub supports deploying HTTPS- and SSL-based services so you ensure your data confidentiality.
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]: Two things:

I was taught that open hyphenation causes translation difficulties, so consider "HTTPS-based and SSL-based services."

I was also taught that there is a legal issue with promising anything outright, including using "ensure" in such a context.

```suggestion
CloudHub supports deploying HTTPS-based and SSL-based services to help you ensure your data confidentiality. 
```

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 5]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-CloudHub supports building HTTPS- and SSL-based services so you can be sure of your data confidentiality. 
-The configuration steps shown in this document are meant for deploying your Mule app to CloudHub.
+CloudHub supports deploying HTTPS- and SSL-based services so you ensure your data confidentiality. 
+These configuration steps shown are meant for deploying your Mule app to CloudHub.
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]:
```suggestion
The following steps illustrate how to configure your Mule app for deployment to CloudHub.
```
steps or tasks?

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 6]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
 
-If you are using Mule 4, you can follow these steps, or take advantage of API Manager integration with secrets manager. 
-Follow the xref:api-manager::building-https-proxy.adoc[Building an HTTPS API Proxy] instructions for more information.
+If you are using:
 
-If you intend to deploy your app locally, see xref:3.8@mule-runtime::tls-configuration.adoc[TLS Configuration]. 
-The xref:deployment-strategies.adoc[Deployment Strategies] documentation offers a closer look at how the configuration should differ between each deployment scenario.
+* *Mule 3 with Anypoint Studio 6*
++
+Follow these steps to build an HTTPS Proxy.
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]:
```suggestion
Follow these steps to build an HTTPS proxy.
```

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 7]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
+* *Mule 4 with Anypoint Studio 7*
++
+Take advantage of Anypoint API Manager integration with the secrets manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]: Does "with" here mean "integration plus secrets" or "integration by using secrets"?

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 8]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
+
+To modify your existing service to use HTTPS:
+
+. Generate a `keystore` file.
 +
-[IMPORTANT]
---
-To generate a self signed certificate now you must specify the hostname in your command, the property 'SAN=DNS:localhost,IP:127.0.0.1' is key in the example below
---
+You can create a `keystore` file using the JDK `keytool` utility on the command line, for example:
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]:
```suggestion
You can create a `keystore` file using the JDK `keytool` utility on the command line:
```

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 9]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
 +
 [source,text,linenums]
 ----
 keytool -genkeypair -keystore keystore.jks   -dname "CN=localhost, OU=Unknown, O=Unknown, L=Unknown, ST=Unknown, C=Unknown"  -keypass password  -storepass password  -keyalg RSA  -sigalg SHA1withRSA  -keysize 2048  -alias mule  -ext SAN=DNS:localhost,IP:127.0.0.1 -validity 9999
 ----
 +
-This command creates a file named `keystore.jks`. Before proceeding, verify that this file exists and appears in the folder `src/main/resources`.
-
-. Configure an HTTPS connector inside your Mule configuration:
+You must specify the hostname (`SAN=DNS:localhost,IP:127.0.0.1` in the example) on the command line to  generate a self-signed certificate.
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]: extra space
```suggestion
You must specify the hostname (`SAN=DNS:localhost,IP:127.0.0.1` in the example) on the command line to generate a self-signed certificate.
```

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 10]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
++
+This command creates a file named `keystore.jks`.
+. Add `keystore.jks` to your Mule app in the folder `src/main/resources` in Studio.
+. Configure an HTTP Connector in your Mule app:
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]: Should this be "Configure Anypoint Connector for HTTP..." 
or "HTTP Connector"
or "an HTTP connector"

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 11]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
+...
+<http:listener-config name="HTTPS_Listener_Configuration" protocol="HTTPS" host="0.0.0.0" port="${https.port}" doc:name="HTTP Listener Configuration">
+     <tls:context>
+         <tls:key-store path="keystore.jks" keyPassword="${keystore.password}" password="password"/>
+     </tls:context>
+ </http:listener-config>
 ...
 ----
-+
+In the HTTP Connector, specify:
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]: ditto connector ref. to actual name?

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 12]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
 
-. Configure any host IP addresses you want to use HTTPS on with the HTTPS scheme and the *`${https.port}`* variable for the port.
- Make sure to include a reference to the HTTPS global connector configuration. For example:
+* Any host IP addresses you want to use with the HTTPS scheme for the value of `host` 
+* `${https.port} variable for the value of `port`
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]: The? An?...variable for...?

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 13]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
+
+. Include a `config-ref` reference to the HTTPS global listener configuration, for example:
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]:
```suggestion
. Include a `config-ref` reference to the HTTPS global listener configuration:
```

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 14]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
+<http:request-config name="HTTP_Request_Configuration" protocol="HTTPS" host="0.0.0.0" port="${https.port}" doc:name="HTTP Request Configuration">
+     <tls:context>
+         <tls:key-store path="keystore.jks" password="password" keyPassword="${keystore.password}" />
+     </tls:context>
+</http:request-config>      
 ...
 ----
 
-These same configurations can all be easily achieved through the Anypoint Studio UI. For instructions on how that is done, and for more in depth configuration details, see the xref:3.8@mule-runtime::http-connector.adoc[HTTP Connector] documentation.
+. Include a `config-ref` reference to the HTTPS global request configuration, for example:
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]:
```suggestion
. Include a `config-ref` reference to the HTTPS global request configuration:
```

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 15]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
++
+[source,xml,linenums]
+----
+<flow name="httpsserviceFlow">
+  <http:request config-ref="HTTP_Request_Configuration" path="some-path" method="GET" doc:name="HTTPS_Request_Configuration" host="0.0.0.0" port="${https.port}"/>
+</flow>
+----
 
-Your application is now ready to be deployed on CloudHub. You can access your endpoint via the normal HTTPS address, for example: `+https://yourdomain.cloudhub.io+`.
+Your application is now ready to be deployed on CloudHub. You can access your endpoint using the HTTPS address, for example: `+https://yourdomain.cloudhub.io+`.
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]:
```suggestion
Your application is now ready to be deployed on CloudHub. You can access your endpoint using the HTTPS address: for example, `+https://yourdomain.cloudhub.io+`.
```

File: modules/ROOT/pages/building-an-https-service.adoc [hunk 16]
```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
...
+
+. Import the proxy project into Studio.
+. Select the *Configuration XML* tab for your proxy flow.
 +
-. Uncomment the http:listener-config block.
-. Fill out the information regarding your keystore: path, pasword and keyPassword.
- You may use external properties. Also keep in mind that the path should not include src/main/resources.
+The parser detects an error because the flow references an HTTPS Connector that is commented out.
+. Uncomment the `http:listener-config` block.
+. Add the `keystore` values: `path`, `password`, and `keyPassword`, for example:
```
Comment (kbostrom) [inline modules/ROOT/pages/building-an-https-service.adoc]:
```suggestion
. Add the `keystore` values: `path`, `password`, and `keyPassword`:
```

**Review by kbostrom — APPROVED** [review]
Voila!

File: modules/ROOT/pages/building-an-https-service.adoc (hunk 10 above)
Comment (hannanelson) [reply modules/ROOT/pages/building-an-https-service.adoc]: Anypoint Connector for HTTP is mentioned earlier. This is a subsequent mention.

**Pull request merged**